packages/uswds-core/src/styles/settings/_settings-utilities.scss

```diff
@@ -618,6 +618,9 @@ $border-manual-values: () !default;
 $border-color-palettes: () !default;
 $border-color-manual-values: () !default;
 
+// .border-high-contrast
+$border-high-contrast: 2px solid transparent;
+
```

Comment:
I'm thinking this setting would be a better fit in `_settings-general`. Maybe we could rename the variable and create a high-contrast section in that file?